core/publish.gradle

```diff
@@ -0,0 +1,77 @@
+apply plugin: "maven-publish"
+apply plugin: "signing"
+
+task sourcesJar(type: Jar, dependsOn: assemble) {
+    classifier = 'sources'
+    from android.sourceSets.main.java.source
+}
+
+// Sonatype will not accept a published artifact without at least an empty Javadoc JAR.
+task javadocsJar(type: Jar, dependsOn: javadocs) {
+    classifier = 'javadoc'
+    from javadocs.destinationDir
+}
+
+afterEvaluate {
+    publishing {
+        repositories {
+            maven {
+                name sonatype.name()
+                url sonatype.url()
+                credentials {
+                    username sonatype.username()
+                    password sonatype.password()
+                }
+            }
+        }
+        publications {
+            release(MavenPublication) {
+                groupId = 'co.infinum'
```

Comment:
We have this in more than one place, right? Why don't we reference it from somewhere?

Reply:
Also - a downside to this approach is that, since we have an additional module for Goldfinger, we want to move to co.infinum.goldfinger anyway to unify its modules. Or com.infinum.goldfinger, for that matter. I'm okay with making the transition here painless for existing users, but in the long term, we should consider fixing this.

Reply:
Yes I agree, in one of a future releases we can do that 👍